unleash-yggdrasil/src/strategy_parsing.rs

```diff
@@ -827,4 +843,42 @@ mod tests {
         let rule = "user_id in [\"Nobody likes \\\"scare quotes\\\"\"]";
         compile_rule(rule).unwrap();
     }
+
+    #[test]
+    fn missing_external_value_produces_false_without_error() {
+        let rule = "external_value[\"i_do_not_exist\"]";
```

Comment:
```
let rule = "!external_value[\"i_do_not_exist\"]";
```
would this be a good test for inverted?

Reply:
I've added specific tests for this. One to check that inversion works on `external_value` and a second to prove that it remains unchanged when its not found (just to match the current state of Ygg)